libid/fractalb.cpp

```diff
@@ -1141,6 +1176,63 @@ BFComplex *cmplx_div_bf(BFComplex *t, BFComplex *x, BFComplex *y)
     return t;
 }
 
+BFComplex *cmplx_poly_bf(BFComplex *out, BFComplex xx, int degree)
```

Comment:
Isn't this function identical to the existing `power` in `complex_fn.h`?

Reply:
Yes, I took cmplx_poly_bf() out of ID. Well spotted.